Engine/Graphics/OpenGL/RenderOpenGL.cpp

```diff
@@ -2856,23 +2853,16 @@ bool RenderOpenGL::MoveTextureToDevice(Texture *texture) {
 }
 
 void _set_3d_projection_matrix() {
-    float near_clip = pIndoorCameraD3D->GetNearClip();
-    float far_clip = pIndoorCameraD3D->GetFarClip();
+    float near_clip = pCamera3D->GetNearClip();
+    float far_clip = pCamera3D->GetFarClip();
 
     glMatrixMode(GL_PROJECTION);
     glLoadIdentity();
 
+    // ogl uses fov in Y so we need to convert
+    float newfov = (180.0 / pi) * 2.0 * atan((game_viewport_height / 2.0) / pCamera3D->ViewPlaneDist_X);
```

Comment:
I'm really lost in the calculations here. Let's just add vertical_fov to pCamera3D?

Reply:
Good idea